docs/BindingsGuide_Java.md

```diff
@@ -1,10 +1,39 @@
 # Azure SQL bindings for Azure Functions - Java
 
+## Setup Function App
+1. Install [Azure Functions Core Tools](https://docs.microsoft.com/azure/azure-functions/functions-run-local)
+
+2. Create a function app for Java:
+    ```bash
+    mkdir MyApp
+    cd MyApp
+    func init --worker-runtime java
+    ```
+
+3. Enable SQL bindings on the function app. More information can be found [in Microsoft Docs](https://docs.microsoft.com/azure/azure-functions/functions-bindings-azure-sql).
+
+    Update the `host.json` file to the preview extension bundle.
+    ```json
+    "extensionBundle": {
+        "id": "Microsoft.Azure.Functions.ExtensionBundle.Preview",
+        "version": "[4.*, 5.0.0)"
+    }
+    ```
+
 ## Input Binding
 
 ### SQLInput Attribute
 
-_TODO_
+In the Java functions runtime library, use the @SQLInput annotation (com.microsoft.azure.functions.sql.annotation.SQLInput) on parameters whose value would come from Azure SQL. This annotation supports the following elements:
+
+| Element |Description|
```

Comment:
I believe this should be Attribute, not Element @lucyzhang929 

```suggestion
| Attribute | Description |
```

Reply:
Looking through the docs - some bindings use Property and some use Element. Looks like Drew used Element for SQL Bindings https://learn.microsoft.com/en-us/azure/azure-functions/functions-bindings-azure-sql-output?tabs=in-process&pivots=programming-language-java#annotations

Reply:
I guess I was wrong - these docs mention "elements" 

https://docs.oracle.com/javase/tutorial/java/annotations/basics.html

`The annotation can include elements, which can be named or unnamed, and there are values for those elements:`

So nevermind then - Element sounds good to me!